homeassistant/components/zwave_me/const.py

```diff
@@ -5,6 +5,6 @@
 # Base component constants
 DOMAIN = "zwave_me"
 
-ZWAVE_PLATFORMS = ["switchMultilevel", "binarySwitch"]
+ZWAVE_PLATFORMS = ["switchMultilevel", "binarySwitch", "toggleButton"]
 
-PLATFORMS = [Platform.NUMBER, Platform.SWITCH]
+PLATFORMS = [Platform.NUMBER, Platform.SWITCH, Platform.BUTTON]
```

Comment:
Minor tweak - we try to keep these in alphabetical order:
```suggestion
PLATFORMS = [Platform.BUTTON, Platform.NUMBER, Platform.SWITCH]
```